backend/src/main/kotlin/fr/gouv/cacem/monitorenv/domain/use_cases/missions/CreateOrUpdateMissionWithAttachedReporting.kt

```diff
@@ -15,6 +15,7 @@ import java.util.UUID
 @UseCase
 class CreateOrUpdateMissionWithAttachedReporting(
```

Comment:
```suggestion
class CreateOrUpdateMissionWithActionsAndAttachedReporting(
```
C'est un peu long comme nom, j'ai mis un peu de temps à comprendre que ce use-case update les reportings et les actions (comparé à l'autre use-case qui n'update que les missions)

Reply:
Oui je suis d'accord. J'ai pas de meilleure idée. peut être isoler dans un autre usecase le rattachement des signalements, et la fonction qui appellerait les trois serait `CreateOrUpdateMissionsWithRelations` ou quelque chose du genre ? des idées ?